src/components/views/settings/SecureBackupPanel.tsx

```diff
@@ -96,6 +96,7 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
 
         MatrixClientPeg.safeGet().on(CryptoEvent.KeyBackupStatus, this.onKeyBackupStatus);
         MatrixClientPeg.safeGet().on(CryptoEvent.KeyBackupSessionsRemaining, this.onKeyBackupSessionsRemaining);
+        MatrixClientPeg.safeGet().on(CryptoEvent.KeyBackupDecryptionKeyCached, this.onKeyBackupDecryptionKeyCached);
```

Comment:
this looks like a functionality change. Please could you either pull it out to a separate PR, or update this PR to emphasise that this is a functionality change with some tests thrown in?